apps/examples/binary_update/binary_update_main.c

```diff
@@ -271,6 +250,8 @@ static void binary_update_run_tests(int repetition_num)
 	binary_info_t cur_bin_info;
 	printf("\n** Binary Update Example %d-th Iteration.\n", repetition_num);
 
+#if CONFIG_FOTA_DELTA_UPDATE == 0
```

Comment:
This is wrong. When `config FOTA_DELTA_UPDATE` is n (not enabled), it is undefined. But your code is working with defined with 0.